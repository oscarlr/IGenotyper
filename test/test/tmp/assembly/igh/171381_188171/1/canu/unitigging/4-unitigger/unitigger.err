COVGAPOLAP 500
LOPSIDED NOBEST 50
MINOLAPPERCENT 0.000000

==> PARAMETERS.

Resources:
  Memory                16 GB
  Compute Threads       4 (command line)

Lengths:
  Minimum read          0 bases
  Minimum overlap       500 bases

Overlap Error Rates:
  Graph                 0.000 (0.030%)
  Max                   0.010 (1.000%)

Deviations:
  Graph                 0.000
  Bubble                3.000
  Repeat                0.000

Similarity Thresholds:
  Graph                 0.000
  Bubble                0.010
  Repeat                0.100

Edge Confusion:
  Absolute              50
  Percent               5.0000

Unitig Construction:
  Minimum intersection  500 bases
  Maxiumum placements   2 positions

Debugging Enabled:
  (none)

==> LOADING AND FILTERING OVERLAPS.

sqStore_loadMetadata()-- Using 'corrected-compressed-trimmed' 0x14 reads.
ReadInfo()-- Using 34 reads, no minimum read length used.

OverlapCache()-- limited to 16384MB memory (user supplied).

OverlapCache()--       0MB for read data.
OverlapCache()--       0MB for best edges.
OverlapCache()--       0MB for tigs.
OverlapCache()--       0MB for tigs - read layouts.
OverlapCache()--       0MB for tigs - error profiles.
OverlapCache()--    4096MB for tigs - error profile overlaps.
OverlapCache()--       0MB for other processes.
OverlapCache()-- ---------
OverlapCache()--    4096MB for data structures (sum of above).
OverlapCache()-- ---------
OverlapCache()--       0MB for overlap store structure.
OverlapCache()--   12287MB for overlap data.
OverlapCache()-- ---------
OverlapCache()--   16384MB allowed.
OverlapCache()--
OverlapCache()-- Retain at least 17 overlaps/read, based on 8.97x coverage.
OverlapCache()-- Initial guess at 23685465 overlaps/read.
OverlapCache()--
OverlapCache()-- Adjusting for sparse overlaps.
OverlapCache()--
OverlapCache()--               reads loading olaps          olaps               memory
OverlapCache()--   olaps/read       all      some          loaded                 free
OverlapCache()--   ----------   -------   -------     ----------- -------     --------
OverlapCache()--      23685465        34         0             618 100.00%      12287 MB
OverlapCache()--
OverlapCache()-- Loading overlaps.
OverlapCache()--
OverlapCache()--          read from store           saved in cache
OverlapCache()--   ------------ ---------   ------------ ---------
OverlapCache()--   ------------ ---------   ------------ ---------
OverlapCache()--            618 (100.00%)            618 (100.00%)
OverlapCache()--
OverlapCache()-- Ignored 0 duplicate overlaps.
OverlapCache()--
OverlapCache()-- Symmetrizing overlaps.
OverlapCache()--   Finding missing twins.
OverlapCache()--   Found 0 overlaps with non-symmetric error rates.
OverlapCache()--   Found 0 missing twins in 618 overlaps, 0 are strong.
OverlapCache()--   Dropping weak non-twin overlaps; allocated 0 MB scratch space.
OverlapCache()--   Dropped 0 overlaps; scratch space released.
OverlapCache()--   Adding 0 missing twin overlaps.
OverlapCache()--   Finished.

BestOverlapGraph()-- Computing Best Overlap Graph.
BestOverlapGraph()-- Allocating best edges (0MB).
BestOverlapGraph()-- Filtering high error edges.
BestOverlapGraph()--   Ignore overlaps with more than 0.001000% error.
BestOverlapGraph()-- Filtering reads with a gap in overlap coverage.
BestOverlapGraph()--   0 reads removed.
BestOverlapGraph()-- Filtering reads with lopsided best edges (more than 50% different).
BestOverlapGraph()--   3 reads have lopsided edges.
BestOverlapGraph()-- Filtering spur reads.
BestOverlapGraph()--   After initial scan, found:
BestOverlapGraph()--         9 spur reads.
BestOverlapGraph()--        10 5' spur paths.
BestOverlapGraph()--         9 3' spur paths.
BestOverlapGraph()--   After iteration 1, found:
BestOverlapGraph()--         9 spur reads.
BestOverlapGraph()--         9 5' spur paths;      2 5' edges changed to avoid a spur path.
BestOverlapGraph()--         9 3' spur paths;      0 5' edges changed to avoid a spur path.
BestOverlapGraph()--   After iteration 2, found:
BestOverlapGraph()--         9 spur reads.
BestOverlapGraph()--         9 5' spur paths;      0 5' edges changed to avoid a spur path.
BestOverlapGraph()--         9 3' spur paths;      0 5' edges changed to avoid a spur path.
Cleared 0 5' and 0 3' best edges on contained reads.

==> BUILDING GREEDY TIGS.

breakSingletonTigs()-- Removed 7 singleton tigs; reads are now unplaced.
optimizePositions()-- Optimizing read positions for 35 reads in 9 tigs, with 4 threads.
optimizePositions()--   Allocating scratch space for 35 reads (1 KB).
optimizePositions()--   Initializing positions with 4 threads.
optimizePositions()--   Recomputing positions, iteration 1, with 4 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged:     34 reads
optimizePositions()--     changed:        1 reads
optimizePositions()--   Recomputing positions, iteration 2, with 4 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged:     34 reads
optimizePositions()--     changed:        1 reads
optimizePositions()--   Recomputing positions, iteration 3, with 4 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged:     34 reads
optimizePositions()--     changed:        1 reads
optimizePositions()--   Recomputing positions, iteration 4, with 4 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged:     34 reads
optimizePositions()--     changed:        1 reads
optimizePositions()--   Recomputing positions, iteration 5, with 4 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged:     34 reads
optimizePositions()--     changed:        1 reads
optimizePositions()--   Expanding short reads with 4 threads.
optimizePositions()--   Updating positions.
optimizePositions()--   Finished.
splitDiscontinuous()-- Tested 1 tig, split none.
detectSpur() done.
tested            1
nEdges      5'    0   3'    0
nPotential        0         0
nVerified         0         0

==> PLACE CONTAINED READS.

computeErrorProfiles()-- Computing error profiles for 9 tigs, with 4 threads.
computeErrorProfiles()-- Finished.

placeContains()-- placing 14 contained and 7 unplaced reads, with 4 threads.
placeContains()-- Placed 13 contained reads and 2 unplaced reads.
placeContains()-- Failed to place 1 contained reads (too high error suspected) and 5 unplaced reads (lack of overlaps suspected).
optimizePositions()-- Optimizing read positions for 35 reads in 9 tigs, with 4 threads.
optimizePositions()--   Allocating scratch space for 35 reads (1 KB).
optimizePositions()--   Initializing positions with 4 threads.
optimizePositions()--   Recomputing positions, iteration 1, with 4 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged:     34 reads
optimizePositions()--     changed:        1 reads
optimizePositions()--   Recomputing positions, iteration 2, with 4 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged:     34 reads
optimizePositions()--     changed:        1 reads
optimizePositions()--   Recomputing positions, iteration 3, with 4 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged:     34 reads
optimizePositions()--     changed:        1 reads
optimizePositions()--   Recomputing positions, iteration 4, with 4 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged:     34 reads
optimizePositions()--     changed:        1 reads
optimizePositions()--   Recomputing positions, iteration 5, with 4 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged:     34 reads
optimizePositions()--     changed:        1 reads
optimizePositions()--   Expanding short reads with 4 threads.
optimizePositions()--   Updating positions.
optimizePositions()--   Finished.
splitDiscontinuous()-- Tested 1 tig, split none.

==> MERGE ORPHANS.

computeErrorProfiles()-- Computing error profiles for 9 tigs, with 4 threads.
computeErrorProfiles()-- Finished.

findPotentialOrphans()-- working on 9 tigs.
findPotentialOrphans()-- found 0 potential orphans.
mergeOrphans()-- flagged       0        bubble tigs with 0 reads
mergeOrphans()-- placed        0 unique orphan tigs with 0 reads
mergeOrphans()-- shattered     0 repeat orphan tigs with 0 reads
mergeOrphans()-- ignored       0               tigs with 0 reads; failed to place
mergeOrphans()--

----------------------------------------
Building new graph after removing 15 placed reads and 0 bubble reads.

BestOverlapGraph()-- Computing Best Overlap Graph.
BestOverlapGraph()-- Allocating best edges (0MB).
BestOverlapGraph()-- Filtering high error edges.
BestOverlapGraph()--   Ignore overlaps with more than 0.001000% error.
BestOverlapGraph()-- Filtering reads with a gap in overlap coverage.
BestOverlapGraph()--   0 reads removed.
BestOverlapGraph()-- Filtering reads with lopsided best edges (more than 50% different).
BestOverlapGraph()--   3 reads have lopsided edges.
BestOverlapGraph()-- Filtering spur reads.
BestOverlapGraph()--   After initial scan, found:
BestOverlapGraph()--         7 spur reads.
BestOverlapGraph()--         6 5' spur paths.
BestOverlapGraph()--         8 3' spur paths.
BestOverlapGraph()--   After iteration 1, found:
BestOverlapGraph()--         7 spur reads.
BestOverlapGraph()--         6 5' spur paths;      0 5' edges changed to avoid a spur path.
BestOverlapGraph()--         8 3' spur paths;      0 5' edges changed to avoid a spur path.
Cleared 0 5' and 0 3' best edges on contained reads.
classifyAsUnassembled()--      0 tigs           0 bases -- singleton
classifyAsUnassembled()--      0 tigs           0 bases -- too few reads        (< 2 reads)
classifyAsUnassembled()--      0 tigs           0 bases -- too short            (< 0 bp)
classifyAsUnassembled()--      0 tigs           0 bases -- single spanning read (> 1.000000 tig length)
classifyAsUnassembled()--      0 tigs           0 bases -- low coverage         (> 0.500000 tig length at < 3 coverage)
classifyAsUnassembled()--      1 tigs       14182 bases -- acceptable contigs


==> GENERATING ASSEMBLY GRAPH.

computeErrorProfiles()-- Computing error profiles for 9 tigs, with 4 threads.
computeErrorProfiles()-- Finished.

AssemblyGraph()-- allocating vectors for placements, 0.002MB
AssemblyGraph()-- finding edges for 28 reads (13 contained), ignoring 6 unplaced reads, with 4 threads.
AssemblyGraph()-- building reverse edges.
AssemblyGraph()-- build complete.

==> BREAK REPEATS.

computeErrorProfiles()-- Computing error profiles for 9 tigs, with 4 threads.
computeErrorProfiles()-- Finished.
markRepeatReads()-- Found 0 confused edges.

==> CLEANUP MISTAKES.

splitDiscontinuous()-- Tested 1 tig, split none.
promoteToSingleton()-- Moved 6 unplaced reads to singleton tigs.
splitDiscontinuous()-- Tested 1 tig, split none.
promoteToSingleton()-- Moved 0 unplaced reads to singleton tigs.

==> CLEANUP GRAPH.


==> GENERATE OUTPUTS.

AssemblyGraph()-- generating './canu.contigs.gfa'.

==> GENERATE UNITIGS.

computeErrorProfiles()-- Computing error profiles for 15 tigs, with 4 threads.
computeErrorProfiles()-- Finished.
splitDiscontinuous()-- Tested 1 tig, split none.
AssemblyGraph()-- generating './canu.unitigs.gfa'.

Bye.
