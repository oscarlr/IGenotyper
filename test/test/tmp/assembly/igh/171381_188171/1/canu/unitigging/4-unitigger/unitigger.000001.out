
Found perl:
   /home/u0jana01/anaconda3/envs/IGv2/bin/perl
   This is perl 5, version 26, subversion 2 (v5.26.2) built for x86_64-linux-thread-multi

Found java:
   /home/u0jana01/anaconda3/envs/IGv2/bin/java
   openjdk version "1.8.0_152-release"

Found canu:
   /home/u0jana01/anaconda3/envs/IGv2/bin/canu
   Canu 2.0

Running job 1 based on command line options.
sqStore_loadMetadata()-- Using 'corrected-compressed-trimmed' 0x14 reads.
/home/u0jana01/IGenotyper/test/test/tmp/assembly/igh/171381_188171/1/canu/unitigging/4-unitigger
/home/u0jana01/IGenotyper/test/test/tmp/assembly/igh/171381_188171/1/canu/unitigging/4-unitigger
